lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context.cpp

```diff
@@ -462,4 +461,33 @@ void amf_free_ue_context(ue_m5gmm_context_s* ue_context_p) {
   ue_context_p = NULL;
 }
 
+// Sync State manager map with Amf Application maps
```

Comment:
Add the `TODO` comment here too for @panyogesh

Reply:
Addressed Review Comment